internal/indexer/definitions/locadora.yaml

```diff
@@ -0,0 +1,56 @@
+---
+#id: Locadora 
+name: Locadora
+identifier: locadora
+description: Locadora is a Brazilian tracker for Movies, TV Shows and Animes.
+language: pt-BR
+urls:
+  - https://locadora.cc/
+privacy: private
+protocol: torrent
+supports:
+  - irc
+    #  - rss
+source: UNIT3D
+settings:
+  - name: rsskey
+    type: secret
+    required: true
+    label: RSS key (RID)
+    help: "Go to your profile tab, Settings > Security, copy RSS Key (RID)"
+
+irc:
+  network: locadora.cc
+  server: irc.locadora.cc
+  port: 6667
+  tls: false
+  channels:
+    - "#announce"
+  announcers:
+    - LCD
+  settings:
+    - name: nick
+      type: text
+      required: true
+      label: Nick
+      help: Bot nick. Eg. user_bot
```

Comment:
To still give the possibility to set a nickserv account from the start add this:

```yaml

    - name: auth.account
      type: text
      required: false
      label: NickServ Account
      help: NickServ account. Make sure to group your user and bot.

    - name: auth.password
      type: secret
      required: false
      label: NickServ Password
      help: NickServ password
```